### ISSUE TITLE
Reproducing Table 1

### ISSUE THREAD

**Issue by berndtt** [atomicarchitects/equiformer_v2#26]
Hi,

Thanks for the amazing work! I tried reproducing the results for the base configuration ($Layers = 12$, $L_{max} = 6$, $M_{max} = 2$) reported in Table 1 (Forces: 20.46, Energy: 285) of the paper using the script `scripts/train/oc20/s2ef/equiformer_v2/equiformer_v2_N@12_L@6_M@2_splits@2M_g@8.sh`. However, I obtained significantly higher values (Forces: 23.31, Energy: 603). Could you confirm whether the script used for the Table 1 results differs from the one currently in the repository?

Many thanks in advance!

**Comment by yilunliao** [comment]
Hi @berndtt 

Thanks for the question.

That one should use the config corresponding to the base model in Table 1.

I have two comments:
1. My initial guess would be the number of GPUs -- we used 16 GPUs (batch size = 64) to obtain the result.
2. In addition, I think you can use the pre-trained checkpoints and run evaluation on validation sub-split (e.g., ID) to double check the pre-processing of the dataset is correct.

Let me know if you can have any other question or can provide more details for this.

**Comment by berndtt** [comment]
Hi @yilunliao,

thanks a lot for you quick reply! It was indeed an error in the data - I had dowloaded the OC20 dataset using raw instead of reference energies. Could you quickly confirm the learning rate you used with a batch size of 64, was it 0.0002 or 0.0004?

**Comment by yilunliao** [comment]
Hi @berndtt 

Great.

Yes, we used: `learning rate = 2e-4, batch size = 64, number of GPUs = 16, training epochs = 12`.

For the one with `training epochs = 30`, we use `learning rate = 4e-4, batch size = 128, number of GPUs = 32`.

**Comment by berndtt** [comment]
Hi @yilunliao,

Thanks for the clarification!
